support-frontend/assets/helpers/forms/existingPaymentMethods/existingPaymentMethods.ts

```diff
@@ -39,7 +39,7 @@ export type ExistingPaymentMethod =
 // ----- Functions ----- //
 function isUsableExistingPaymentMethod(
 	existingPaymentMethod: ExistingPaymentMethod,
-): boolean {
+): existingPaymentMethod is RecentlySignedInExistingPaymentMethod {
```

Comment:
by using a type guard here typescript can then handle the type cast whilst using this function in a call to `.filter`!